### ISSUE TITLE
InteractableColorTheme does not support Image

### ISSUE THREAD

**Issue by gilbdev** [microsoft/MixedRealityToolkit-Unity#6628]
## Describe the bug
Similar to #4551, when using the theme for an "UI/Image" component (inside a canvas) and having in interactable with a theme targeting this image,
I got the following exception :
```
NullReferenceException: Object reference not set to an instance of an object
Microsoft.MixedReality.Toolkit.UI.InteractableColorTheme.TryGetRendererColor (Microsoft.MixedReality.Toolkit.UI.ThemeStateProperty property, UnityEngine.Color& color) (at Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/VisualThemes/ThemeEngines/InteractableColorTheme.cs:238)
```

## To reproduce

1. Create an interactable with a canvas, and a sub gameobject with an image
2. Add a profile and a color theme targeting the Image
3. Play

## Expected behavior

Color theme suppports Canvas Image

## Your Setup (please complete the following information)

- Unity Version [2018.4.9f1]
- MRTK Version [v2.1]

**Comment by Troy-Ferrell** [comment]
I believe this is fixed for 2.2 as it works for me in stabilization. @gilbdev, can you please check your settings to see if anything is out of place or different in your setup to below? If you also have time, update to 2.2 to see if your situation is fixed.

![image](https://user-images.githubusercontent.com/25975362/69175890-7bc5a780-0ab9-11ea-9e30-c6dda41a284c.png)

**Comment by gilbdev** [comment]
indeed, this seems to have been addressed in 2.2 !
